src/models/group.model.ts

```diff
@@ -0,0 +1,19 @@
+import {DataTypes} from 'sequelize';
+
+export const GroupModel = {
+  id: {
+    type: DataTypes.UUID,
+    primaryKey: true,
+    allowNull: false,
+    unique: true,
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  permissions: {
+    type: DataTypes.ARRAY(
+      DataTypes.ENUM('READ', 'WRITE', 'DELETE', 'SHARE', 'UPLOAD_FILES')
```

Comment:
you can use enum to define the permissions in this file and reuse them in API
https://www.typescriptlang.org/docs/handbook/enums.html#string-enums